test/extractors/saarland_pdf_extractor_originators_test.rb

```diff
@@ -12,7 +12,7 @@ class SaarlandPDFExtractorOriginatorsTest < ActiveSupport::TestCase
   end
 
   test 'one person, single line 2' do
-    paper = Struct.new(:contents).new("Anfrage der Abgeordneten Astrid Schramm (DIE LINKE.) \n")
+    paper = Struct.new(:contents).new("Anfrage der Abgeordneten Astrid Schramm (DIE LINKE.)")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.